client/landing/gutenboarding/stores/onboard/persist.ts

```diff
@@ -48,7 +48,7 @@ const isNotExpired = ( timestampStr: string ): boolean => {
 
 // Check for "fresh" query param
 const hasFreshParam = (): boolean => {
-	return window.location.search.indexOf( 'fresh' ) !== -1;
+	return new URLSearchParams( window.location.search ).has( 'fresh' );
```

Comment:
Just need some input - are we worried about IE support for this? Looks like we're using it a couple of other places, but idk.

Reply:
I think [our use of corejs 3](https://github.com/Automattic/wp-calypso/blob/a47b3fc421244477d0cc577112dcc1763b52f810/packages/calypso-build/babel/default.js#L19) covers that as a polyfill but I'm not actually sure about `wp-polyfill` in WP admin. 🤔 

https://github.com/babel/babel/issues/10452